src/main/java/com/sysc4806app/model/User.java

```diff
@@ -27,8 +30,16 @@ public User(){
         following = new ArrayList<>();
     }
 
+    public User(String name, String password) {
+        this.name = name;
+        this.password = password;
+        reviews = new ArrayList<>();
+        following = new ArrayList<>();
+    }
+
     public User(String name) {
         this.name = name;
+        this.password = "pass1";
```

Comment:
I could be persuaded but I feel like we shouldn't allow this. I think we should make a password mandatory and later we can add validation for users to ensure passwords are of a certain length/complexity.

Reply:
I just kept this contructor so i wouldn't have to rewrite tests, I'll fix it an remove this